t/unit/transport/test_redis.py

```diff
@@ -1353,9 +1356,10 @@ def test_getting_master_from_sentinel(self):
             connection.channel()
             patched.assert_called_once_with(
                 [
+                    [u'localhost', u'65532'],
+                    [u'localhost', u'65533'],
                     [u'localhost', u'65534'],
                     [u'localhost', u'65535'],
-                    [u'localhost', u'65536']
```

Comment:
Good catch as `2^16-1` is `65,535`

Reply:
Yeah `_parse_url()` (or `urllib.parse` to be exact) was actually throwing an exception because of it.

```python
>>> _parse_url('sentinel://:password@localhost:65536/')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/peter/src/github.com/celery/kombu/kombu/utils/url.py", line 58, in url_to_parts
    parts.port,
  File "/usr/local/Cellar/python/3.7.2_2/Frameworks/Python.framework/Versions/3.7/lib/python3.7/urllib/parse.py", line 171, in port
    raise ValueError("Port out of range 0-65535")
ValueError: Port out of range 0-65535
```